webpack.config.js

```diff
@@ -0,0 +1,65 @@
+const path = require('path');
+const WasmPackPlugin = require('@wasm-tool/wasm-pack-plugin');
```

Comment:
Is this required? We are using Trunk already so Im not sure why we have a WASM loader here.